tests/ui/components/filters/filteroptionscomponent.js

```diff
@@ -657,4 +658,42 @@ describe('filter options component', () => {
       expect(() => COMPONENT_MANAGER.create('FilterOptions', config)).toThrow();
     });
   });
+
+  describe('interaction with global storage', () => {
+    let globalStorage;
+    let COMPONENT_MANAGER;
+
+    beforeEach(() => {
+      const mockCore = {
+        setStaticFilterNodes: setStaticFilterNodes,
+        setLocationRadiusFilterNode: () => { },
+        filterRegistry: {
+          setStaticFilterNodes: setStaticFilterNodes
+        },
+        globalStorage: new GlobalStorage(),
+        persistentStorage: new PersistentStorage()
+      };
+
+      COMPONENT_MANAGER = mockManager(mockCore);
+
+      defaultConfig = {
+        container: '#test-component',
+        control: 'singleoption',
+        options: options,
+        label: filterOptionsLabel
+      };
+
+      globalStorage = COMPONENT_MANAGER.core.globalStorage;
+    });
+
+    it('deletes the global storage FilterOptions key when constructed', () => {
```

Comment:
could we update this to check it deletes the storage key equal to the component name you give it through config?
would it be okay to get rid of the extra line breaks too?